src/dataflow/src/decode/regex.rs

```diff
@@ -12,13 +12,13 @@ use std::str;
 
 use differential_dataflow::{AsCollection, Collection};
 use regex::Regex;
-use timely::dataflow::operators::{OkErr, Operator};
+use timely::dataflow::operators::Operator;
 use timely::dataflow::{Scope, Stream};
 
 use dataflow_types::{DataflowError, DecodeError};
 use repr::{Datum, Diff, Row, Timestamp};
 
-use crate::source::SourceOutput;
+use crate::{operator::CollectionExt, source::SourceOutput};
```

Comment:
Set the "Import Merge Behavior" setting in your editor to "last".

(I personally don't care at all, but if you don't, sooner or later someone who _does_ care about import formatting will ask you to :) )

Reply:
What does this setting do? Changing the setting does not appear to cause a diff in rust analyzer?

Reply:
If you had had the setting on when you automatically added the import, it wouldn't have merged these.

Unfortunately it doesn't retroactively affect already-merged use statements, so you will have to fix those by hand, if you care (up to you!)

Reply:
Oh, I see now! I'll unmerge them then!

We do see to have quite a few other examples of nested merges, now that I know what I'm looking for. Thanks!